app/graphql/queries/fetch_dog.rb

```diff
@@ -0,0 +1,16 @@
+module Queries
+  class FetchDog < Queries::BaseQuery
+    type Types::DogType, null: false
+    argument :id, ID, required: true
+
+    def resolve(id:)
+      # binding.pry
```

Comment:
do we want to remove the prys before merging to main so we don't have to refactor later?

Reply:
That's my bad! Good catch. Sometimes I let an errant pry slip through.